packages/sampler/stories/default/Layout.js

```diff
@@ -21,28 +21,33 @@ export const _Layout = (args) => (
 			orientation={args['orientation']}
 		>
 			<Cell
-				size={args['cell size'] + 'px'}
+				size={args['first cell size'] + 'px'}
 				shrink
 			>
-				<Item>First</Item>
+				<Item>First (shrink)</Item>
```

Comment:
I'm not sure but it seems that adding "(shrink)" affects its height when the size is small.
🤔 Should we change the content?

Reply:
I'll add `NO-BREAK SPACE`, which is &#xa0; between First and (shrink).